src/components/forms/formr-part-b/viewSections/ViewSectionTestHelper.tsx

```diff
@@ -0,0 +1,75 @@
+// load type definitions that come with Cypress module
+/// <reference types="cypress" />
+import {
+  FormRPartB,
+  Work,
+  Declaration,
+  CovidDeclaration
+} from "../../../../models/FormRPartB";
+import { DateUtilities } from "../../../../utilities/DateUtilities";
+import { BooleanUtilities } from "../../../../utilities/BooleanUtilities";
+
+type FormRPartBKeys = keyof FormRPartB;
+type FormRPartBWorkKeys = keyof Work;
+type FormRPartBDeclarationKeys = keyof Declaration;
+type FormRPartBCovidDeclarationKeys = keyof CovidDeclaration;
+
+interface ISectionDataField {
+  fieldName: FormRPartBKeys;
+  format: string;
+}
+
+interface ISectionWorkDataField {
+  fieldName: FormRPartBWorkKeys;
+  format: string;
+}
+
+interface ISectionDeclarationDataField {
+  fieldName: FormRPartBDeclarationKeys;
+  format: string;
+}
+
+interface ISectionCovidDeclarationDataField {
+  fieldName: FormRPartBCovidDeclarationKeys;
+  format: string;
+}
+
+export interface ISectionSomeDataField {
+  fieldName:
+    | FormRPartBKeys
+    | FormRPartBWorkKeys
+    | FormRPartBDeclarationKeys
+    | FormRPartBCovidDeclarationKeys;
+  format: string;
+}
+
+export default function CheckDataIsDisplayed(
+  formDataToDisplay: ISectionSomeDataField[],
```

Comment:
Johno suggested something like 
`export function genericThing<T>(formDataToDisplay: T[], formData:any){formDataToDisplay.forEach((formDataItem: any) => {…`
Might be worth exploring!